deploy/vagrant/vbox/buildbox.sh

```diff
@@ -12,9 +14,16 @@ if [ -f tachyon-dev.box ]; then
   rm -f tachyon-dev.box
 fi
 
-vagrant up
-vagrant package --output tachyon-dev.box default
-vagrant destroy -f
+BASE_BOX='chef/centos-7.0'
+AVAIL_BOX=$(vagrant box list | grep $BASE_BOX | grep '(virtualbox, 1.0.0)')
+
```

Comment:
@yuzhu I agree with you that in the current design where we try to save a box with java/maven installed, your suggestion makes sense. But I propose we take care of the tachyon-dev box saving as a separate task. There are two principles I think we should stick to. First, we should let users know what's going to happen regarding tachyon-dev box creation instead of hiding a "vagrant up" in another "vagrant up".  This also allows user to choose different caching strategy -- most often developers will use the same ufs and tachyon to develop, so saving the downloading time will be helpful. Second, we should use ansible to provision all things.

Having that said, I am going to update this PR by reverting the build_vbox.sh to old version, and use your suggested fix to replace my fix. But I would still like to keep one change I made -- keep two "vagrant up" in a sequential way instead of nested way. So this will complete this PR.

Also, I will create a Jira ticket on how to improve the box packaging strategies to facilitate both user experiences and developer's development needs. Most likely, when user call "create", they have several options: a) use vanilla vm -- this will be cleaner and more extensible on other images. b)  package the box with minimal environment -- java, maven etc c) package the box with everything -- improve developer's working efficiency.  We will also create a re-launch script.

What do you think?